react/src/components/organisms/Footer/Footer.stories.js

```diff
@@ -13,7 +13,7 @@ import SocialLinksLiveData from '../../molecules/SocialLinks/SocialLinksLive.jso
 import stateSeal from '../../../../../assets/images/stateseal.png';
 
 
-storiesOf('organisms/Footer', module).addDecorator(withKnobs)
```

Comment:
same as above, the Footer stories were previously grouped under organisms, now cluttering up the listing:
![screen shot 2018-09-13 at 16 21 37](https://user-images.githubusercontent.com/38226003/45519149-24b79a80-b771-11e8-93e3-4b1c10882b7a.png)